python/ray/autoscaler/_private/event_system.py

```diff
@@ -87,7 +163,7 @@ def execute_callback(
         if event_data is None:
             event_data = {}
 
-        event_data["event_name"] = event
+        event_data["event"] = event
```

Comment:
Was this change intentional/necessary? Seems like this would break backwards compatibility with existing callbacks.

Reply:
It was intentional but not necessary. I figured `"event_name"` is a misleading name since it points to an Enum with a state sequence ID and event name. For example the distinction between `d['event_name'].name` and `d['event_name'].value` seems a bit vague.